src/test/java/api/support/builders/TimeZoneConfigBuilder.java

```diff
@@ -0,0 +1,38 @@
+package api.support.builders;
+
+import io.vertx.core.json.JsonObject;
+
+public class TimeZoneConfigBuilder extends JsonBuilder implements Builder {
```

Comment:
Is there anything that makes this specific to time zone configuration?

It seems to me this only reflects the general structure of a configuration record?

Reply:
Yes, that's right, this class keeps the record structure for configurations.
Fixed